### PR TITLE
Use wildcard for ingress backend. Ensures /v1/health etc work correctly

### PR DESCRIPTION
**Pull request by AndrewCsoka**
@tuannvm @mlaccetti Without this, I found hitting /v1/health would return `404 - default backend` when using an Ingress

### CODE REVIEW

**Review by mlaccetti — APPROVED** [review]
/approve

**Comment by AndrewCsoka** [comment]
I think this is waiting for approval from @lachie83

Is there anything else that need's to be done?

**Comment by AndrewCsoka** [comment]
@mlaccetti  Any ideas why this isn't progressing please? The healthcheck is broken without this change

**Comment by mlaccetti** [comment]
@mattfarina @unguiculus I think this is waiting for one of you guys to type the magic slash command. 😁

**Comment by mattfarina** [comment]
This PR was submitted when circleci testing was broken (tests ran but did not report back) and under the CLI (prior to the DCO). Re-ran the pull-charts-e2e which made sure it's working so merging manually.

**Pull request merged**
